extensions/sql-migration/src/constants/helper.ts

```diff
@@ -29,8 +105,8 @@ export function getMigrationTargetType(migration: DatabaseMigration): string {
 	return '';
 }
 
-export function getMigrationTargetTypeEnum(migration: DatabaseMigration): MigrationTargetType | undefined {
-	switch (migration.type) {
+export function getMigrationTargetTypeEnum(migration: DatabaseMigration | undefined): MigrationTargetType | undefined {
```

Comment:
what happens if undefined is passed in?

Reply:
also just curious, why is this needed?

Reply:
the code that calls this function needs to be able to pass in undefined likely based on the source of the parameter used... the code handles undefined in the default: section.